README.md

```diff
@@ -12,6 +12,10 @@ EVE supports both ARM and Intel architectures and requires hardware-assisted vir
 
 To get its job done, EVE leverages a lot of great open source projects: [Xen Project](https://xenproject.org/), [Linuxkit](https://github.com/linuxkit/linuxkit) and [Alpine Linux](https://alpinelinux.org/) just to name a few. All of that functionality is being orchestrated by the Go microservices available under [pkg/pillar](pkg/pillar). Why pillar? Well, because pillar is the kind of a monolith we need to break out into true, individual microservices under [pkg/](pkg/).
 
+## Download EVE
+
+EVE LTS: [Donwload latest LTS](https://github.com/lf-edge/eve/releases/latest)
+
```

Comment:
Doesn't make sense to also put a link for all releases (non-LTS)? I mean, a link to https://github.com/lf-edge/eve/releases page

Reply:
Yes, make sence, but we using pre-relase. 
For Pre-relase github dont apply semantic version sorting. Its means old but recently patched version can be latest in that case. Plus, link as **latest** for release section can be only one.

Reply:
ok, got it.